src/test/regress/expected/drop_partitioned_table.out

```diff
@@ -0,0 +1,469 @@
+(0 rows)
+
+\c - - - :master_port
+SET search_path = drop_partitioned_table;
+SET citus.next_shard_id TO 720000;
+-- CASE 4
+-- DROP OWNED BY role1; Parent and children are owned by role1
+GRANT ALL ON SCHEMA drop_partitioned_table TO role1;
+SET ROLE role1;
+CREATE TABLE drop_partitioned_table.parent (x text, t timestamptz DEFAULT now()) PARTITION BY RANGE (t);
+CREATE TABLE drop_partitioned_table.child1 (x text, t timestamptz DEFAULT now());
+ALTER TABLE drop_partitioned_table.parent ATTACH PARTITION drop_partitioned_table.child1 FOR VALUES FROM ('2021-05-31') TO ('2021-06-01');
+CREATE TABLE drop_partitioned_table.child2 (x text, t timestamptz DEFAULT now());
+ALTER TABLE drop_partitioned_table.parent ATTACH PARTITION drop_partitioned_table.child2 FOR VALUES FROM ('2021-06-30') TO ('2021-07-01');
+RESET ROLE;
+SELECT create_distributed_table('parent','x');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+\d
+                      List of relations
+         Schema         |  Name  |       Type        | Owner
+---------------------------------------------------------------------
+ drop_partitioned_table | child1 | table             | role1
+ drop_partitioned_table | child2 | table             | role1
+ drop_partitioned_table | parent | partitioned table | role1
+(3 rows)
+
+SELECT logicalrelid, shardid FROM pg_dist_shard WHERE shardid >= 720000 AND shardid < 730000 ORDER BY shardid;
+ logicalrelid | shardid
+---------------------------------------------------------------------
+ parent       |  720000
+ parent       |  720001
+ parent       |  720002
+ parent       |  720003
+ child1       |  720004
+ child1       |  720005
+ child1       |  720006
+ child1       |  720007
+ child2       |  720008
+ child2       |  720009
+ child2       |  720010
+ child2       |  720011
+(12 rows)
+
+\set VERBOSITY terse
+DROP OWNED BY role1;
+NOTICE:  dropped object: drop_partitioned_table.parent original: t normal: f
+NOTICE:  dropped object: drop_partitioned_table.child1 original: t normal: f
+NOTICE:  dropped object: drop_partitioned_table.child2 original: t normal: f
+\d
+      List of relations
+ Schema | Name | Type | Owner
+---------------------------------------------------------------------
+(0 rows)
+
+SELECT logicalrelid, shardid FROM pg_dist_shard WHERE shardid >= 720000 AND shardid < 730000 ORDER BY shardid;
+ logicalrelid | shardid
+---------------------------------------------------------------------
+(0 rows)
+
+\c - - - :worker_1_port
+SET search_path = drop_partitioned_table;
+\d
+      List of relations
+ Schema | Name | Type | Owner
+---------------------------------------------------------------------
+(0 rows)
+
+\c - - - :master_port
+SET search_path = drop_partitioned_table;
+SET citus.next_shard_id TO 720000;
+REVOKE ALL ON SCHEMA drop_partitioned_table FROM role1;
+DROP ROLE role1;
+SELECT run_command_on_workers('DROP ROLE role1');
```

Comment:
would this test fail on enterprise?

Reply:
Yes, thanks for noticing. Adding `IF EXISTS`.